ui/src/message_popup/final_summary_card.jsx

```diff
@@ -10,19 +10,15 @@ export default React.createClass({
 
   render(){
     const mean = Math.round(_.mean(this.props.responseTimes)/10)/100;
-    const numUnderTime = _.filter(this.props.responseTimes, function(time){
-      return time <= this.props.limitMs;
-    }.bind(this)).length;
-    const numAboveTime = _.filter(this.props.responseTimes, function(time){
-      return time > this.props.limitMs;
-    }.bind(this)).length;
+    const numUnderTime = this.props.responseTimes.filter(time => time <= this.props.limitMs).length;
+    const numAboveTime = this.props.responseTimes.filter(time => time > this.props.limitMs).length;
```

Comment:
You can also use `_.partition` for this if you like: https://lodash.com/docs#partition